### DISCUSSION TITLE
Tábua de marés

### DISCUSSION THREAD

**Discussion by jlucas577** [BrasilAPI/BrasilAPI#228] [Ideas]
Poderiamos aproveitar o desenvolvimento dessa API, e adicionar um endpoint para obter os dados das tábuas das marés de uma determinada região, atualmente é necessário ter um código para ler o PDF que a marinha fornece em seu site.

**Comment by enieber** [comment]
@jlucas577 tem mais informa sobre isso?

**Comment by enieber** [comment]
@jlucas577 pesquisando aqui verifique que existe [um serviço  que faz algo parecido](https://tabuademares.com/br), não  sei como eles fizeram, mas seria interessante descobrir para implementar, outra questão, para quem se interessar por essa issue, tem o site da marinha com alguns dados: https://www.marinha.mil.br/chm/tabuas-de-mare
